litestar/channels/backends/redis.py

```diff
@@ -149,6 +174,7 @@ def __init__(
         self._stream_ttl = stream_ttl if isinstance(stream_ttl, int) else int(stream_ttl.total_seconds() * 1000)
         self._flush_all_streams_script = self._redis.register_script(_FLUSHALL_STREAMS_SCRIPT)
         self._publish_script = self._redis.register_script(_XADD_EXPIRE_SCRIPT)
+        self._has_subscribed_channels = _LazyEvent()
```

Comment:
```suggestion
        self._has_subscribed = _LazyEvent()
```
Nit, but this is consistent with the pub/sub backend.

Reply:
Could this be assigned in the super class instead of in each subclass?

Reply:
Nit on the nit: they represent slightly different things. For the pubsub this guards the `PubSub.subscribed` property, and here it’s whether the channel count is positive.

Although conceptually they’re the same.

Reply:
> Could this be assigned in the super class instead of in each subclass?

I thought about handling the `_LazyEvent` thing in the superclass, but it felt nicer this way. I’ll sleep on it.

Reply:
well don't lose sleep on it - all good either way